plugins/release-manager-as-a-service/src/cards/createRc/CreateRc.tsx

```diff
@@ -0,0 +1,205 @@
+import {
+  ComponentConfigCreateRc,
+  GhGetBranchResponse,
+  GhGetReleaseResponse,
+  GhGetRepositoryResponse,
+  Project,
+  SetRefetch,
+} from '../../types/types';
+import { ResponseStepList } from '../../components/ResponseStepList/ResponseStepList';
+import { useStyles } from '../../styles/styles';
+import { useApiClientContext } from '../../components/ProjectContext';
+import { SEMVER_PARTS } from '../../constants/constants';
+import { TEST_IDS } from '../../test-helpers/test-ids';
+
+interface CreateRcProps {
+  defaultBranch: GhGetRepositoryResponse['default_branch'];
+  latestRelease: GhGetReleaseResponse | null;
+  project: Project;
+  releaseBranch: GhGetBranchResponse | null;
+  setRefetch: SetRefetch;
+  successCb?: ComponentConfigCreateRc['successCb'];
+}
+
+export const CreateRc = ({
+  defaultBranch,
+  latestRelease,
+  project,
+  releaseBranch,
+  setRefetch,
+  successCb,
+}: CreateRcProps) => {
+  const apiClient = useApiClientContext();
+  const classes = useStyles();
+
+  const [semverBumpLevel, setSemverBumpLevel] = useState<'major' | 'minor'>(
+    SEMVER_PARTS.minor,
+  );
+  const [nextGheInfo, setNextGheInfo] = useState(
+    getRcGheInfo({ latestRelease, project, semverBumpLevel }),
+  );
+
+  useEffect(() => {
+    setNextGheInfo(getRcGheInfo({ latestRelease, project, semverBumpLevel }));
+  }, [semverBumpLevel, setNextGheInfo, latestRelease, project]);
+
+  const [createReleaseResponse, callCreateGheRc] = useAsyncFn(
+    async (...args) => {
+      const createGheRcResponseSteps = await createGheRc({
+        apiClient,
+        defaultBranch,
+        latestRelease,
+        nextGheInfo: args[0],
+        successCb,
+      });
+
+      return createGheRcResponseSteps;
+    },
+  );
+
+  if (createReleaseResponse.error) {
+    return (
+      <Alert severity="error">{createReleaseResponse.error.message}</Alert>
+    );
+  }
+
+  const tagAlreadyExists =
+    latestRelease !== null &&
+    latestRelease.tag_name === nextGheInfo.rcReleaseTag;
+  const conflictingPreRelease =
+    latestRelease !== null && latestRelease.prerelease;
+
+  function Description() {
+    if (conflictingPreRelease) {
+      return (
+        <Alert className={classes.paragraph} severity="warning">
+          The most recent release is already a Release Candidate
+        </Alert>
+      );
+    }
+
+    if (tagAlreadyExists) {
+      return (
+        <Alert className={classes.paragraph} severity="warning">
+          There's already a tag named{' '}
+          <strong>{nextGheInfo.rcReleaseTag}</strong>
+        </Alert>
+      );
+    }
+
+    return (
+      <div className={classes.paragraph}>
+        <Typography>
+          <Differ
+            icon="branch"
+            prev={releaseBranch?.name}
+            next={nextGheInfo.rcBranch}
+          />
+        </Typography>
+
+        <Typography>
+          <Differ
+            icon="tag"
+            prev={latestRelease?.tag_name}
+            next={nextGheInfo.rcReleaseTag}
```

Comment:
To repeat my feedback re: GitHub.com vs. GHE, if this can support both, may be useful to refactor any vars that say GHE?